app/services/service_provider_seeder.rb

```diff
@@ -13,7 +13,8 @@ def run
                   active: true,
                   native: true,
                   friendly_name: config['friendly_name'])
-      end.update!(config.except('restrict_to_deploy_env',
+      end.update!(config.except('agency',
```

Comment:
this means there's no urgency to updating identity-idp-config to remove the string agency attribute